pkg/config/optimizely_config.go

```diff
@@ -23,12 +23,15 @@ import (
 
 // OptimizelyConfig is a snapshot of the experiments and features in the project config
 type OptimizelyConfig struct {
-	Revision       string                          `json:"revision"`
-	ExperimentsMap map[string]OptimizelyExperiment `json:"experimentsMap"`
-	FeaturesMap    map[string]OptimizelyFeature    `json:"featuresMap"`
-	SdkKey         string                          `json:"sdkKey,omitempty"`
-	EnvironmentKey string                          `json:"environmentKey,omitempty"`
-	datafile       string
+	Revision            string                          `json:"revision"`
+	ExperimentsMap      map[string]OptimizelyExperiment `json:"experimentsMap"`
+	FeaturesMap         map[string]OptimizelyFeature    `json:"featuresMap"`
```

Comment:
OptimizleyFeature missing "ExperimentRules" and "DeiliveryRules" new fields. 
I see most FSC failures come from these missing fields.